xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -58,7 +58,8 @@ To be able to format the output with indentation and white space OR to skip vali
         <param name="options">Defines the customized behavior of the <see cref="T:System.Text.Json.Utf8JsonWriter" /> By default, it writes minimized JSON (with no extra whitespace) and validates that the JSON being written is structurally valid according to the JSON RFC.</param>
         <summary>Constructs a new <see cref="T:System.Text.Json.Utf8JsonWriter" /> instance with a specified <see langword="System.Buffers.IBufferWriter&lt;System.Byte&gt;" />.</summary>
         <remarks>To be added.</remarks>
-        <exception cref="T:System.ArgumentNullException"><paramref name="bufferWriter" />  is <see langword="null" />.</exception>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="bufferWriter" />  is <see langword="null" />.</exception>
```

Comment:
This looks like it has extra whitespace characters.
```suggestion
          <paramref name="bufferWriter" /> is <see langword="null" />.</exception>
```

Reply:
But, I don't see that change in this PR anymore, but we should fix it in master:
https://github.com/dotnet/dotnet-api-docs/blob/9eea13dfeaec8478af2fb08239628ce430a80f25/xml/System.Text.Json/Utf8JsonWriter.xml#L62

Reply:
I've removed some of the changes because the CI process would undo that again. PR opened to fix the space issue: https://github.com/dotnet/dotnet-api-docs/pull/3152